### PR TITLE
pounce: 1.1 -> 1.3p1

### PR DESCRIPTION
**Pull request by glittershark**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/pounce/default.nix [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 
-  buildInputs = [ libressl ];
-
-  configurePhase = "ln -s Linux.mk config.mk";
+  buildInputs = [ libressl pkg-config ];
 
   buildFlags = [ "all" ];
 
   makeFlags = [
     "PREFIX=$(out)"
```
Comment (causal-agent) [inline pkgs/servers/pounce/default.nix:17]: Might be preferable to now set `DESTDIR` and `PREFIX` separately, although it doesn't matter to pounce.

**Comment by causal-agent** [comment]
Apparently as the author of pounce I should leave my seal of approval on this, so

LGTM :)

File: pkgs/servers/pounce/default.nix [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
   src = fetchzip {
     url = "https://git.causal.agency/pounce/snapshot/pounce-${version}.tar.gz";
-    sha256 = "07iyh6ikrlf7y57k462jcr00db6aijk9b2s7n7l7i49hk7kmm6wq";
+    sha256 = "1ab4pz7gyvlms00hcarcmsljkn0whwqxfck8b343l4riai2rj9xv";
   };
 
-  buildInputs = [ libressl ];
-
-  configurePhase = "ln -s Linux.mk config.mk";
+  buildInputs = [ libressl pkg-config ];
```
Comment (danieldk) [inline pkgs/servers/pounce/default.nix:12]: `pkg-config` should go into `nativeBuildInputs`.

```suggestion
    nativeBuildInputs = [ pkg-config ];
  
    buildInputs = [ libressl ];
```

**Review by danieldk — CHANGES_REQUESTED** [review]

File: pkgs/servers/pounce/default.nix (hunk 2 above)
Comment (glittershark) [reply pkgs/servers/pounce/default.nix]: done.

**Review by danieldk — APPROVED** [review]
LGTM

Result of `nixpkgs-review pr 92726` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- pounce
</details>

**Pull request merged**